YarnSpinner.LanguageServer.Tests/TestData/TestWorkspace/Project1/ExampleCommands.cs

```diff
@@ -69,4 +69,13 @@ public static void CommandWithComplexDocs()
     {
 
     }
+
+    static class NestingTestParentClass
+    {
+     
+        [YarnCommand("command_nested_in_parent_class")]
+        public static void CommandNestedInParentClass()
+        {
+        }
+    }
```

Comment:
It's not the static-ness that's causing the problem, it's that `CommandNestedInParentClass` is being added to the `taggedMethods` dictionary for every class it is a descendent of (ie first for `ExampleCommands` then for `NestingTestParentClass`). (Was worried for a minute there that C# CodeAnalysis was doing something like variable hoisting in Javascript but thankfully not 😅)